README.md

```diff
@@ -241,6 +241,32 @@ This SDK exposes the following classes:
 - `TransactionsWidget`
 - `TrendsWidget`
 
+### Troubleshooting
+
+#### Post messages not working
+
+Check the following items if you have passed post message callbacks in your
+widget instantiation but they are not being triggered:
+
+- Ensure your SSO request is correctly configured for the Web SDK:
+    - `ui_message_version` should be set to `4`.
+    - `is_mobile_webview` should be set to `true`.
+- Ensure you the appropriate widget class for the `widget_type` used when
+  making the SSO request. For example, if you set `widget_type` to be
+  `connect_widget`, then you should use the `ConnectWidget` class expected by
+  the SDK.
+- Ensure you are serving your application's HTML file from a web server and not
+  loading it from your file system.
+- Check your application's [`Referrer-Policy`][referrer_policy]. Due to the
+  nature of post message, we recommend using the default Referrer Policy, which
+  is `strict-origin-when-cross-origin`.
+
+  If you are unable to use the default policy, then you may use
```

Comment:
I know `target_origin_referrer` is new and not documented, we could leave this paragraph our for the time being but it does make sense that it would be included somewhere that is helping users fix their post message issues.